.env-dist

```diff
@@ -22,6 +22,11 @@ SES_CONFIG_SET=""
 # 1: only log messages coming back from SES
 SES_NOTIFICATION_LOG_ONLY=
 
+# Firefox newsletter
+BASKET_URL=https://basket.allizom.org
+BASKET_API_KEY=
+BASKET_NEWLSETTER=mozilla-and-you
```

Comment:
typo on "NEWLSETTER".

Reply:
Also, is this the correct newsletter, or do we have some specific Monitor one we're creating?